### PR TITLE
Feature#6/sign up UI

### LINKED ISSUE #6: Create Sign Up UI.
-Create tittle 
-Create Text Inputs
-Create check boxes 
-Create buttons 
-Create Log In footer

### PR DESCRIPTION
**Pull request by JacintoOrtiz15745**
# Description
What did you implemented/Why did you implemented this:
-We implemented the UI Design of screen SignUp
 
 # Testing
NA

  
  ## Screenshots 
Screenshot_1610564149

### CODE REVIEW

**Comment by HRuvalcabaP** [comment]
I think it's okay!

File: ReservacionDeVuelos/src/components/atoms/SignUpTitle.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React, {useEffect} from 'react';
+import {Text, Button, TextInput} from 'react-native';
+import {styles} from '../../styles/SignUpStyles';
```
Comment (HRuvalcabaP) [inline ReservacionDeVuelos/src/components/atoms/SignUpTitle.js]: i think that is good job to import styles from other Js

**Review by HRuvalcabaP — DISMISSED** [review]

**Comment by vmagana3** [comment]
It looks ok!!,  you respect the proposed design, Just the last text "Login" is not aligned correctly. I would changed that!!

File: ReservacionDeVuelos/src/components/atoms/SignUpButtons.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+          style={[styles.googleIcon, {marginLeft: -75}]}
+        />
+        <Text style={[styles.btnSignupText, {marginLeft: 50}]}>
+          Sign Up with Google
+        </Text>
+      </TouchableOpacity>
+      <Text style={styles.loginTextContainer}>
+        Already have an account?{' '}
+        <TouchableOpacity>
+          <Text style={styles.LogInText}> Log In</Text>
```
Comment (vmagana3) [inline ReservacionDeVuelos/src/components/atoms/SignUpButtons.js]: I would aligned this text!!

**Review by vmagana3 — DISMISSED** [review]

**Comment by sebasdrk17** [comment]
It looks okay but i have a few suggestions, in the password input you should display an icon of an eye, but overall is good!

**Review by sebasdrk17 — DISMISSED** [review]
Looks great!

File: ReservacionDeVuelos/src/components/atoms/SignUpForm.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    <View>
+      <Text style={styles.formText}>First Name</Text>
+      <TextInput style={styles.textInput} />
+      <Text style={styles.formText}>Email *</Text>
+      <TextInput style={styles.textInput} />
+      <Text style={styles.formText}>Password *</Text>
+      <TextInput style={styles.textInput} secureTextEntry={true} />
+      <Text style={styles.characters}>
+        Use 8 or more characters with a mix of letters, numbers, and sumbols
+      </Text>
```
Comment (LeyaimMagma) [inline ReservacionDeVuelos/src/components/atoms/SignUpForm.js]: can you refactor this?

File: ReservacionDeVuelos/src/styles/SignUpStyles/index.js [hunk 4]
```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import {StyleSheet} from 'react-native';
+
+export const styles = StyleSheet.create({
+  title: {
+    fontSize: 20,
+    color: '#2D02FF',
```
Comment (LeyaimMagma) [inline ReservacionDeVuelos/src/styles/SignUpStyles/index.js]: create a _colors.js_ file to add all the colors that will be used in the app

**Review by LeyaimMagma — CHANGES_REQUESTED** [review]

File: ReservacionDeVuelos/src/components/atoms/SignUpButtons.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+  return (
+    <View style={styles.btnSignupContainer}>
+      <TouchableOpacity style={styles.btnSignup}>
+        <Text style={styles.btnSignupText}>Sign Up</Text>
+      </TouchableOpacity>
+      <Text style={styles.or}>or</Text>
+      <TouchableOpacity style={styles.btnSignup}>
+        <Image
+          source={GoogleIcon}
+          style={[styles.googleIcon, {marginLeft: -75}]}
```
Comment (Ing-Heber) [inline ReservacionDeVuelos/src/components/atoms/SignUpButtons.js]: I would Align the google icon too.

**Review by Ing-Heber — CHANGES_REQUESTED** [review]
Nice job, but you have to  align the google icon, the log in link and also look for a way to save or refactor some code in signUptitle.js

File: ReservacionDeVuelos/src/components/atoms/SignUpButtons.js (hunk 5 above)
Comment (Joaquin-Niembro) [reply ReservacionDeVuelos/src/components/atoms/SignUpButtons.js]: Yes! i´ll work on those changes!

File: ReservacionDeVuelos/src/components/atoms/SignUpForm.js (hunk 3 above)
Comment (Joaquin-Niembro) [reply ReservacionDeVuelos/src/components/atoms/SignUpForm.js]: Yes! I´ll work on that rigth away!

**Review by JordiGN — DISMISSED** [review]
Good job in general, just the same comments as the other mentors/bright coders, but you are on fire!!! Keep working like that

**Comment by Joaquin-Niembro** [comment]
i think changes have been made with success! 
kata

**Review by fernandoamz — CHANGES_REQUESTED** [review]
Add all the string in the constants file all call it where you need it

**Review by Ing-Heber — DISMISSED** [review]
Niceeeeeeeeee!!, here are some tips that will help you in future PR documentation:

1. When possible add the design view so mentors and reviewers can see how your design has to look like.
2. On the Testing section you can write a detailed description for reviewers of how to execute your changes in reviewers local.
3. When you update your screen design remember to update the screenshot so we can see the change.
Those tips helped me so much and made my work a little bit nicer jeje, nice job Jacinto

**Review by JordiGN — DISMISSED** [review]

**Review by fernandoamz — CHANGES_REQUESTED** [review]
Create the snapshoots testing for all your components

File: ReservacionDeVuelos/src/styles/Colors.js [hunk 6]
```diff
@@ -0,0 +1,5 @@
+export const colors = {
+    Purple: '#2D02FF',
+    ligthGray: '#b6b7ba',
+    white: '#FFF'
+}
```
Comment (LeyaimMagma) [inline ReservacionDeVuelos/src/styles/Colors.js:6]: Add blank space here

**Review by LeyaimMagma — CHANGES_REQUESTED** [review]

**Review by tovl1 — DISMISSED** [review]
Nice!

**Review by Ing-Heber — DISMISSED** [review]

**Review by fernandoamz — DISMISSED** [review]
Only a little recommendation, Try to install enzyme and create the snapshots with enzyme. Try to do it in your next pull request

**Comment by Joaquin-Niembro** [comment]
> Only a little recommendation, Try to install enzyme and create the snapshots with enzyme. Try to do it in your next pull request

Yes! WE will use enzyme! Thanks!

File: ReservacionDeVuelos/src/styles/Colors.js (hunk 6 above)
Comment (Joaquin-Niembro) [reply ReservacionDeVuelos/src/styles/Colors.js:6]: Could you give us more details?
Do you mean blank space like BlankSpace: ' ' ? or something else?

**Review by LeyaimMagma — APPROVED** [review]

**Review by Joaquin-Niembro — APPROVED** [review]

**Review by jjcr11 — APPROVED** [review]

**Pull request merged**